src/CGRA.cpp

```diff
@@ -36,7 +36,7 @@ CGRA::CGRA(int t_rows, int t_columns, bool t_diagonalVectorization,
       }
     }
 
-    ifstream paramCGRA("./paramCGRA.json");
+    ifstream paramCGRA("./param.json");
```

Comment:
Would this change make our previous ICED or CGRA-Flow compilation fail on "file not found"?